components/Button.js

```diff
@@ -0,0 +1,10 @@
+import Link from "next/link";
+import { HiArrowLeft } from "react-icons/hi2";
+
+export default function Button() {
```

Comment:
This seems to be a back button. Just naming it "Button" might be confusing, I think.